charts/opentelemetry-demo-datadogv2/collector-exporter.yaml

```diff
@@ -0,0 +1,47 @@
+opentelemetry-demo:
+  default:
+    envOverrides:
+      - name: HOST_IP
+        valueFrom:
+          fieldRef:
+            fieldPath: status.hostIP
+      - name: OTEL_RESOURCE_ATTRIBUTES
+        value: >-
+          service.name=$(OTEL_SERVICE_NAME),
+          k8s.namespace.name=$(OTEL_K8S_NAMESPACE),
+          k8s.node.name=$(OTEL_K8S_NODE_NAME),
+          k8s.pod.name=$(OTEL_K8S_POD_NAME),
+          deployment.environment=opentelemetry-demo-collector
```

Comment:
technically only this last line is changed between collector & agent but I think there is no way to yaml-fu that :/